tst/testinstall/attribute.tst

```diff
@@ -70,5 +76,13 @@ gap> FavouriteFruit(foo);
 "pear"
 gap> HasFavouriteFruit(foo);
 true
+gap> SetFavouriteFruit(foo, "apple");
+#I  Attribute FavouriteFruit of <object> already set to pear, cannot be changed to apple
```

Comment:
should there perhaps be a similar test for *mutable* attributes which verifies that no warning is printed, and that the value is changed successfully?